src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -28,6 +28,11 @@
     private Enemy contains;
 
     private long timeContained = 0;
+    
+    private long timeEmpty = 0;
+    
+    private final long TIME_FLOAT_UPWARDS = 50;
```

Comment:
Unread field: website.frontrow.board.Bubble.TIME_FLOAT_UPWARDS; should this field be static?[FindBugs]

<bug